go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/bytebase/bytebase
 go 1.18
 
 require (
-	github.com/ClickHouse/clickhouse-go/v2 v2.0.7
+	github.com/ClickHouse/clickhouse-go/v2 v2.2.0
```

Comment:
If we only need to upgrade clickhouse, then just do that instead of upgrading other dependencies.

Reply:
Let's take a try.

Reply:
I ran `go get -u github.com/ClickHouse/clickhouse-go/v2`, and other packages also got upgraded.